### ISSUE TITLE
EG.5.1.1>Orf1a:T4129I>S:L455F recombinant with Orf10 of XBB.1.16(23 seq, 4 countries)

### ISSUE THREAD

**Issue by Over-There-Is** [sars-cov-2-variants/lineage-proposals#1098]
EG.5.1.1> C12651T(Orf1a:T4129I)> G22927T(S:L455F)> C7504T> C29386T, T29625Crev(Orf10:F23Srev)
Breakpoint: 28447\~29386
5' to 28447: EG.5.1.1> C12651T(Orf1a:T4129I)> G22927T(S:L455F)> C7504T (9 seq, South Korea, all with additional C723T)
29386 to 3': XBB.1.16

Query: C7504T, C12651T, C29386T

Earliest sequence: 09-11 South Korea EPI_ISL_18354239
Latest sequence: 12-23 South Korea EPI_ISL_18793140
Regions Detected: South Korea 8, US 3 (California 1, NY 1, Texas 1), China 2(Taiwan), Germany 1(Brandenburg)

https://nextstrain.org/fetch/github.com/Over-There-Is/subtreeAuspice/raw/main/subtreeAuspice_genome_33d9f_5fb280.json?label=id:node_3521713
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/38100637/f499b023-df55-4cb4-abb2-636e4a37cb09)

Sequences:
EPI_ISL_18354239, EPI_ISL_18439365, EPI_ISL_18440672,
EPI_ISL_18511537, EPI_ISL_18520817, EPI_ISL_18539400,
EPI_ISL_18546729, EPI_ISL_18551093, EPI_ISL_18564745,
EPI_ISL_18564930, EPI_ISL_18597700, EPI_ISL_18599956,
EPI_ISL_18620591, EPI_ISL_18712269, EPI_ISL_18792028-18792033,
EPI_ISL_18792180, EPI_ISL_18792187, EPI_ISL_18793140

**Comment by Over-There-Is** [comment]
South Korea+1

**Comment by Over-There-Is** [comment]
US+2, CA & TX

**Comment by Over-There-Is** [comment]
KR+2

**Comment by FedeGueli** [comment]
it seems fast.

**Comment by Over-There-Is** [comment]
Brandenburg, Deutschland

**Comment by Over-There-Is** [comment]
NY, US

**Comment by FedeGueli** [comment]
No new sequence in December closing for now.

**Comment by Over-There-Is** [comment]
But it is detected in the latest batch of sequences from South Korea.

**Comment by FedeGueli** [comment]
> C7504T, C12651T, C29386T

Yeah but still no new sequence from December , pls ping me if more than one will be found.

**Comment by Over-There-Is** [comment]
Korea +9, with the latest sequence collected on 12-23, reopen it

**Comment by FedeGueli** [comment]
> The delay for South Korean sequences being uploaded is quite long, always about a month.

yeah noticed that too. agree.

**Comment by Over-There-Is** [comment]
South_Korea/KDCA281712/2023|EPI_ISL_18792033|2023-12-18 Potential recombined with BA.2.86*, T22480Crev, I332V, T346Rrev, K356T
